asapdiscovery-alchemy/asapdiscovery/alchemy/schema/fec.py

```diff
@@ -225,9 +325,14 @@ class _FreeEnergyBase(_SchemaBase):
         MultiStateSimulationSettings(
             equilibration_length=1.0 * OFFUnit.nanoseconds,
             production_length=5.0 * OFFUnit.nanoseconds,
+            time_per_iteration=2.5 * OFFUnit.picoseconds,
         ),
         description="Settings for simulation control, including lengths and writing to disk.",
     )
+    adaptive_settings: AdaptiveSettings = Field(
+        AdaptiveSettings(),
```

Comment:
field should be optional with default None, and then the adaptive_sampling field on AdaptiveSampling class should go away.